amplipi/models.py

```diff
@@ -474,6 +474,8 @@ class Stream(Base):
   client_id: Optional[str] = Field(description='Plexamp client_id, becomes "identifier" in server.json')
   token: Optional[str] = Field(description='Plexamp token for server.json')
   server: Optional[str] = Field(description='Server url')
+  index: Optional[int] = Field(description='RCA index')
+  disbaled: Optional[bool] = Field(description="Soft disable use of this stream. It won't be shown as a selectable option")
```

Comment:
```suggestion
  disabled: Optional[bool] = Field(description="Soft disable use of this stream. It won't be shown as a selectable option")
```

Reply:
I'm going to assume this means either disabling wasn't tested or this field is useless?

Reply:
LoL it worked before! I think unused fields are just unvalidated?